src/lib.rs

```diff
@@ -16,39 +36,57 @@ pub mod arch;
 #[panic_handler]
 fn panic_handler(info: &core::panic::PanicInfo) -> ! {
     use esp_println::println;
+    
+    #[cfg(not(feature = "print-defmt"))]
```

Comment:
just an idea: maybe having a function for RED and RESET (including the cfg-guards) would make things a bit easier to read

Reply:
Yes, made code more readable, thanks!
Done.